definition.go

```diff
@@ -0,0 +1,88 @@
+package supervisor
+
+import (
+	"fmt"
+	"go.uber.org/zap"
+	"strings"
+	"time"
+)
+
+// Definition is the configuration for process to supervise
+type Definition struct {
+	Command                []string `json:"command,omitempty"`
+	Replicas               int
```

Comment:
Make sure all user-facing fields have JSON struct tags. Also add godoc comments for everything user-facing, this will automatically be part of generated documentation on the Caddy website when you register your module. It's how people will learn how to use your module.